server/services/store/sqlstore/board.go

```diff
@@ -763,3 +772,256 @@ func (s *SQLStore) getBoardMemberHistory(db sq.BaseRunner, boardID, userID strin
 
 	return memberHistory, nil
 }
+
+func (s *SQLStore) GetBoardsForUserAndTeam(userID, teamID string, includePublicBoards bool) ([]*model.Board, error) {
```

Comment:
Methods in the store that appear in the store interface should be private methods that first receive a `sq.BaseRunner` parameter so they're reusable within the store. Running `make generate` will create duplicated entries for all these methods in the `public_methods.go` file

Reply:
Yup, nice catch.